py/picca/cf.py

```diff
@@ -425,7 +425,7 @@ def x_forest_cf1d(pix):
             we2 = d2.we
             xi1d[bins] += wde1 * wde2[:,None]
             we1d[bins] += we1*we2[:,None]
-            nb1d[bins] += 1
+            nb1d[bins] += (we1*we2[:,None]>0.).astype(int)
 
```

Comment:
We remove masked bins from the forests, don't we? There shouldn't be any pixels with zero weight.